modules/LemDCBM400600/utils/lem_dcbm_api_mock/main.py

```diff
@@ -0,0 +1,383 @@
+                    "energyImportTotalStop": 196.614,
+                    "energyExport": 0.000,
+                    "energyExportTotalStart": 0.000,
+                    "energyExportTotalStop": 0.000
+                }},
+            "meterId": "12024072805",
+            "signature": "304502203DC38FBC722D216568D6ECB4B3"
+                         "52577A999B6D184EA6AD48BDCAE7766DB1D628022100A768"
+                         "7B4CB5573829D407DD4B17D41C297917B7E8307E5017711B"
+                         "5A3A987F6801",
+            "publicKey": "A80F10D968E1122F8820F288B23C4E1C0D"
+                         "A912F35B48481274ADFEFE66D7E87E130C7CF2B8047C45CF"
+                         "105041C8C3A57DD242782F755C9443F42DABA9404A67BF"
+        }
+    )
+
+
+class OCMFReading(BaseModel):
+    TM: str  # actually datetime + status
+    TX: str
+    RV: float
+    RI: str
+    RU: str
+    RT: str
+    EF: str
+    ST: str
+    UC: dict  # not in standard? LEM : "This field reflects the /settings/cableConf selected table for the transaction by the /legal/ cableId input parameter. This is a LEM specific field, using specific IDs:"
+
+
+class OCMFPart1(BaseModel):
+    FV: str
+    GI: str
+    GS: str
+    GV: str
+    PG: str
+    MV: str
+    MM: str
+    MS: str
+    MF: str
+    IS: bool
+    IL: str
+    IF: list[str]
+    IT: str
+    ID: str
+    TT: str
+    RD: list[OCMFReading]
+
+
+OCMF_EXAMPLE_JSON1 = {
+    "FV": "1.0",
+    "GI": "ABL SBC-301",
+    "GS": "808829900001",
+    "GV": "1.4p3",
+    "PG": "T12345",
+    "MV": "Phoenix Contact",
+    "MM": "EEM-350-D-MCB",
+    "MS": "BQ27400330016",
+    "MF": "1.0",
+    "IS": True,
+    "IL": "VERIFIED",
+    "IF": [
+        "RFID_PLAIN",
+        "OCPP_RS_TLS"
+    ],
+    "IT": "ISO14443",
+    "ID": "1F2D3A4F5506C7",
+    "TT": "Tarif 1",
+    "RD": [
+        {
+            "TM": "2018-07-24T13:22:04,000+0200 S",
+            "TX": "B",
+            "RV": 2935.6,
+            "RI": "1-b:1.8.0",
+            "RU": "kWh",
+            "RT": "AC",
+            "EF": "",
+            "ST": "G",
+            # LEM Special
+            "UC": {"UN": "cableName", "UI": 1, "UR": 1}
+        }
+    ]
+}
+
+
+class OCMFPart2(BaseModel):
+    SA: str = Field(
+        required=False)  # NOTE: could not find this in https://github.com/SAFE-eV/OCMF-Open-Charge-Metering-Format/blob/master/OCMF-en.md, but specified in LEM
+    SD: str
+
+
+OCMF_EXAMPLE_JSON2 = {
+    "SA": "ECDSA-secp256r1-SHA256",  # LEM special
+    "SD": "887FABF407AC82782EEFFF2220C2F856AEB0BC22364BBCC6B55761911ED651D1A922BADA88818C9671AFEE7094D7F536"
+}
+
+
+@v1_api.get("/ocmf", response_class=PlainTextResponse)
+def get_last_transaction_ocmf(transactionId: str | None) -> str:
+    return f"OCMF|{OCMFPart1(**OCMF_EXAMPLE_JSON1)}|{OCMFPart2(**OCMF_EXAMPLE_JSON2)}"
+
+
+@v1_api.get("/ocmf/{transactionIndex}", response_class=PlainTextResponse)
+def get_last_transaction_ocmf(transactionIndex: int) -> str:
```

Comment:
Can you have a look if that's a real issue or a false positive?

Reply:
Was a real issue, resolved by renaming the function.